superset/assets/cypress/utils/readResponseBlob.js

```diff
@@ -0,0 +1,11 @@
+// This function returns a promise that resolves to the value
+// of the passed response blob. It assumes the blob should be read as text,
+// and that the response can be parsed as JSON. This is needed to read
+// the value of any fetch-based response.
+export default function readResponseBlob(blob) {
```

Comment:
Can you double check cypress doesn't add this in as a test? If you run tests just make sure this file doesn't show up. I had trouble adding helpers / utils because it kept picking them up as tests. I'm not sure if it will happen because this is outside of the integration folder, but if it does I added a setting to ignore any files with the pattern `*.helper.js`.

Reply:
@michellethomas thanks for bringing this up, it looks okay from the cypress UI

<img src="https://user-images.githubusercontent.com/4496521/45732244-3ee9e200-bb90-11e8-898d-eaad40d90ba3.png" width="200" />